phpunit/class-wp-theme-json-test.php

```diff
@@ -1099,12 +1044,12 @@ public function test_get_stylesheet_with_deprecated_feature_level_selectors() {
 			)
 		);
 
```

Comment:
If I'm reading the comments above this test and the next one correctly, we should be able to update them to use core blocks now?

Reply:
That's my understanding. We should be able to update these tests to core blocks and remove the filter registering the test block from the phpunit bootstrap.

I can look a little deeper on this tomorrow if that helps?

Reply:
Thanks for confirming, I'll give it a go and see if it works!

Reply:
Does the "deprecated" in the test name refer to defining selectors with `__experimentalSelector` inside the `supports` array (like Calendar or Button do) as opposed to outside (like Image does)?

Reply:
I believe so, it looks like this test was renamed from `test_get_stylesheet_with_block_support_feature_level_selectors` in https://github.com/WordPress/gutenberg/pull/46496/ which stabilised that feature and moved it outside of `supports`.

Reply:
The selectors API feature got merged by a good samaritan before it was actually ready so following threads like #46486 will be tricky as the individual PRs only contain part of the "story".

To answer the question, yes, the deprecated `__experimentalSelector` properties are what the test was about as we have to maintain backwards compatibility for them unfortunately.

Reply:
Thanks folks, all done! I trimmed the properties output to match what each block supports, and that also makes the strings a more manageable size.